sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -67,6 +67,7 @@
     # Local configurations
     IntegrationTestRepoConfig(),
     IntegrationTestRepoConfig(python_feature_server=True),
+    IntegrationTestRepoConfig(online_store=REDIS_CONFIG, go_feature_server=True),
```

Comment:
this should be included in the `DEFAULT_FULL_REPO_CONFIGS.extend(` part

the baseline `DEFAULT_FULL_REPO_CONFIGS` should only contain configurations that can be tested without launching infra; a repo config using Redis doesn't belong here